buildpacks/nodejs-npm-install/README.md

```diff
@@ -8,7 +8,7 @@ Heroku's official Cloud Native Buildpack for executing `npm install`.
 
 The buildpack will pass detection if:
  
-- A `package-lock.json` file is found at the root of the application source.
+- A `package.json` file is found at the root of the application source.
```

Comment:
Doesn't that mean this buildpack will be detected and run in `pnpm` or `yarn` scenarios?

Reply:
No, because it's in the [last group](https://github.com/heroku/buildpacks-nodejs/blob/main/meta-buildpacks/nodejs/buildpack.toml#L43-L61) that we check in the composite node buildpack. The first ordered group that matches should be chosen, right? So this will only come into play if the pnpm and Yarn groups fail detection.

Reply:
> No, because it's in the [last group](https://github.com/heroku/buildpacks-nodejs/blob/main/meta-buildpacks/nodejs/buildpack.toml#L43-L61) that we check in the composite node buildpack. The first ordered group that matches should be chosen, right? So this will only come into play if the pnpm and Yarn groups fail detection.

Yes, that's right. Folks using the buildpack outside of the meta-buildpack could potentially see the issue, though.